extensions/amp-analytics/amp-analytics.md

```diff
@@ -521,6 +523,8 @@ In the example below, `beacon` and `xhrpost` are set to `false`, so they will no
 }
 ```
 
+If the `iframe` transport option is used, the specified URL must point to a page which implements the iframe transport client API. See the [example](https://github.com/ampproject/amphtml/blob/master/examples/analytics-iframe-transport-remote-frame.html) of how to use this API, and also an [example](https://github.com/ampproject/amphtml/blob/master/examples/analytics-iframe-transport.amp.html) of a page that incorporates that iframe. The example loads a [fake ad](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad-network-fake-impl/0.1/data/fake_amp_ad_with_iframe_transport.html), which contains the `amp-analytics` tag. Note that the fake ad content includes some extra configuration instructions that must be followed. 
```

Comment:
Consider rewriting "See ..." so the links are a wee bit more specific, like so:

`To learn more, see [this example that implements iframe transport client API](https://github.com/ampproject/amphtml/blob/master/examples/analytics-iframe-transport-remote-frame.html) and [this example page that incorporates that iframe](https://github.com/ampproject/amphtml/blob/master/examples/analytics-iframe-transport.amp.html). The example page ...`